nex/register_common_secure_server_protocols.go

```diff
@@ -0,0 +1,10 @@
+package nex
+
+import (
+	"github.com/PretendoNetwork/friends-secure/globals"
+	secureconnection "github.com/PretendoNetwork/nex-protocols-common-go/secure-connection"
+)
+
+func registerCommonSecureServerProtocols() {
+	secureconnection.NewCommonSecureConnectionProtocol(globals.SecureServer)
```

Comment:
The common protocols don't implement `RegisterEx`, so we have to override it here

```suggestion
	secureconnection.NewCommonSecureConnectionProtocol(globals.SecureServer)
	
	secureconnection.RegisterEx(nex_secure_connection.RegisterEx)
```

With this we can also remove the `nex/secure-connection/register.go` from here (it isn't used by friends anyway)

Reply:
Added override, removed unneeded file